src/systems/systems/src/hydraulic/brake_circuit.rs

```diff
@@ -471,19 +471,14 @@ impl AutobrakeDecelerationGovernor {
         if self.is_engaged {
             self.time_engaged += context.delta();
 
-            let target_error = self.filtered_acceleration.get::<meter_per_second_squared>()
-                - self.target.get::<meter_per_second_squared>();
-
-            let p_term = self.p_gain * (target_error - self.last_error);
-            let i_term = self.i_gain * target_error;
-            self.current_output += p_term + i_term;
-
-            self.last_error = target_error;
-
-            self.current_output = self.current_output.min(1.).max(0.);
+            self.current_output = -self.pid_controller.next_control_output(
+                self.filtered_acceleration.get::<meter_per_second_squared>(),
+                Some(context.delta()),
+            );
```

Comment:
**question**: Do you expect situations in which the delta wouldn't be available (and thus would be `None`)? Right now this only happens in your unit tests. Outside of unit tests it is always `Some`.

Reply:
The pid library that Blueberry used at first had the "flaw" of not handling those delta times. 
So current implementation can use them, or not if None.

It should be used without delta in a fixed step context only (or adjusting gains depending on delta...).